package.json

```diff
@@ -152,6 +152,11 @@
       "pretendToBeVisual": true
     }
   },
+  "standard-version": {
+    "scripts": {
+      "postchangelog": "npm run prettify"
```

Comment:
I'm still wondering if prettify, for code at least, should be run before build and test, just to make sure it doesn't mangle something that it shouldn't (and the tests should pick it up).

Part of the issue would be that the build (which is what gets published), should be prettified before?

It could be run twice... once before build, and once after standard-version makes/updates the change log file. I think prettier supports passing a file name so the changelog portion could just target the changelog file.

Note we still have to manually edit the changelog to correct the heading levels and heading names (for the sub headings), until we can find a way to pass a custom template to `standard-version`

Currently `standard-version` only accepts `preset` names, where as the underlying `conventional-changelog` command _does_ accept custom templates.

Reply:
See https://github.com/conventional-changelog/standard-version/issues/250